puppet_module/build_module.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+
+# Wazuh package generator
+# Copyright (C) 2022, Wazuh Inc.
```

Comment:
If new development, it should be 2023

Reply:
Done: 8ff1104df0b5b451cbc8267df4e72a9c60c28fd0